<!--

    Copyright (C) 2012-2024 SonarSource SA - mailto:info AT sonarsource DOT com
    This code is released under [MIT No Attribution](https://opensource.org/licenses/MIT-0) license.

-->
<p>My stupid rule to avoid extends some SuperClass</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO 
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO 
</pre>
